certhandler/modules/pkcs11module/pkcs11module.go

```diff
@@ -85,6 +85,8 @@ type PKCS11Module struct {
 	userPIN     string
 	tokenLabel  string
 	pendingKeys *list.List
+	ctxMutex    sync.Mutex
```

Comment:
It should be global. It covers case when you have many pkcs11 modules in one application. It is our case.